pkg/devfile/detect.go

```diff
@@ -252,10 +252,11 @@ func AnalyzePath(a Alizer, localpath, context, devfileRegistryURL string, devfil
 	if !isDevfilePresent && isDockerfilePresent {
 		// Still invoke alizer to detect the ports from the component
 		_, _, _, detectedPorts, err := AnalyzeAndDetectDevfile(a, localpath, devfileRegistryURL)
-		if err != nil {
-			return err
+		if err == nil {
+			componentPortsMapFromRepo[context] = detectedPorts
+		} else {
+			log.Info("failed to detect port from context: %v, error: %v", context, err)
```

Comment:
should this be formatted?  `log.Info(fmt.Sprintf("failed to detect port from context: %v, error: %v", context, err))` or  alternatively follow the key/value pair logging convention for that API.

Reply:
oops. updated